dandisets/dandiset_000004_converted.yaml

```diff
@@ -0,0 +1,240 @@
+    Berkeley, CA, USA
+  - Department of Neurosurgery, Stanford University, Stanford, CA, USA
+  email: ben.dichter@gmail.com
+  name: Dichter, Ben
+  roleName:
+  - Author
+  - Software
+  - ProjectMember
+  - Validation
+- affiliation:
+  - Institute for Interdisciplinary Brain and Behavioral Sciences, Crean College of
+    Health and Behavioral Sciences, Schmid College of Science and Technology, Chapman
+    University, Orange, CA, USA
+  - Division of Biology and Biological Engineering, California Institute of Technology,
+    Pasadena, CA, USA
+  email: maoz.uri@gmail.com
+  name: Maoz, Uri
+  roleName:
+  - Author
+  - Conceptualization
+  - ProjectMember
+  - Validation
+- affiliation:
+  - Division of Neurosurgery, Department of Surgery, University of Toronto, Toronto,
+    Canada
+  - Krembil Brain Institute, Toronto Western Hospital, Toronto, Canada
+  email: suneil.kalia@uhn.ca
+  name: Kalia, Suneil K.
+  roleName:
+  - Author
+  - ProjectMember
+  - Validation
+- affiliation:
+  - Krembil Brain Institute, Toronto Western Hospital, Toronto, Canada
+  - Division of Neurosurgery, Department of Surgery, University of Toronto, Toronto,
+    Canada
+  email: Taufik.Valiante@uhn.ca
+  name: Valiante, Taufik A.
+  roleName:
+  - Author
+  - ProjectMember
+  - Validation
+- affiliation:
+  - Department of Neurosurgery, Cedars-Sinai Medical Center, Los Angeles, CA, USA
+  email: Adam.Mamelak@cshs.org
+  name: Mamelak, Adam N.
+  roleName:
+  - Author
+  - ProjectMember
+  - Validation
+- affiliation:
+  - Department of Neurosurgery, Cedars-Sinai Medical Center, Los Angeles, CA, USA
+  - Department of Neurology, Cedars-Sinai Medical Center, Los Angeles, CA, USA
+  - Division of Biology and Biological Engineering, California Institute of Technology,
+    Pasadena, CA, USA
+  - Computational and Neural Systems Program, California Institute of Technology,
+    Pasadena, CA, USA
+  - Center for Neural Science and Medicine, Department of Biomedical Science, Cedars-Sinai
+    Medical Center, Los Angeles, CA, USA
+  email: Ueli.Rutishauser@cshs.org
+  identifier:
+    propertyID: ORCID
+    value: 0000-0002-9207-7069
+  name: Rutishauser, Ueli
+  roleName:
+  - Author
+  - Conceptualization
+  - FundingAcquisition
+  - ProjectMember
+  - Resources
+  - Software
+  - Supervision
+  - Validation
+- awardNumber: U01NS103792
+  name: Stroke, National Institute of Neurological Disorders and
+  roleName: &id001
+  - Sponsor
+- awardNumber: '1554105'
+  name: Foundation, National Science
+  roleName: *id001
+- awardNumber: R01MH110831
+  name: Health, National Institute of Mental
+  roleName: *id001
+- name: Neuroscience, McKnight Endowment for
+  roleName: *id001
+- name: Foundation, NARSAD Young Investigator grant from the Brain & Behavior Research
+  roleName: *id001
+- name: Foundation, Kavli
+  roleName: *id001
+- awardNumber: U19NS104590
+  name: initiative, BRAIN
+  roleName: *id001
```

Comment:
A couple of things:

* I'm not sure what the value of`roleName` is supposed to represent in these? `*id001` for example?
* These don't match either of the schema that they're required to ([either `Person` or `Organization`](https://github.com/satra/dandi-core/blob/90b3b84f5258f9b7a449bc9e997133574c1b2fc7/scripts/models.py#L119-L124)). I suspect these are supposed to represent organizations, in which case they're missing the `contactPoint` field.

I'm not sure how many other files this applies to, just noticed in when testing and wanted to use this data as an example.

Reply:
@AlmightyYakob - that's how yaml does references. you'll notice the first one shows 
```
  roleName: &id001
  - Sponsor
```
and then that rolename is referred to in the remaining roleNames. this is just yaml reference syntax. a yaml parser should be able to load this just fine.

regarding your second question all `roleName` options are in `roleType` the schema should have them. yes, `contactPoint` is missing for all of them because it wasn't there. knowing that some of these orgs could be funding institutions. we can make it potentially optional. i'll change it in the model and the schema.